test/Garnet.test/RespTests.cs

```diff
@@ -1898,7 +1898,6 @@ public void HelloTest1()
             Assert.AreEqual(2, (int)resultDict["proto"]);
```

Comment:
Might want to add at least one basic test for ASYNC and we can expand on it later...

Reply:
Added test